docs/index.rst

```diff
@@ -108,6 +99,7 @@ Mission Statement
     development/index
     CHANGELOG.md
     roadmap
+    Code Quality Guidelines <Code_Quality_Guidelines.rst>
```

Comment:
This should go in `development/index.rst` so as to put in "Developer Workflow" like I mentioned earlier. We don't want to increase items in sidebar - it should only have high-level items.